commands/config.go

```diff
@@ -0,0 +1,47 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/BurntSushi/toml"
+	"github.com/bndw/pick/backends"
+	"github.com/bndw/pick/crypto"
+	"github.com/mitchellh/go-homedir"
+)
+
+var (
+	config                Config // holds the global pick config.
+	defaultConfigFileTmpl = "%s/.pick/pick.toml"
```

Comment:
Let's roll with this for now. I think it'd nice to accept a `--config` flag which would likely make this obsolete or at least revisit it.